bentoml/_internal/bento/build_dev_bentoml_whl.py

```diff
@@ -28,11 +28,27 @@ def build_bentoml_editable_wheel(target_path: str) -> None:
         # skip this entirely if BentoML is installed from PyPI
         return
 
+    try:
+        from build.env import IsolatedEnvBuilder
+
+        from build import ProjectBuilder
+    except ModuleNotFoundError as e:
+        raise MissingDependencyException(_exc_message) from e
+
     # Find bentoml module path
     module_location = source_locations("bentoml")
     if not module_location:
         raise BentoMLException("Could not find bentoml module location.")
 
+    try:
+        from importlib import import_module
+
+        _ = import_module("bentoml.grpc.v1.service_pb2")
```

Comment:
What's the difference between this and `import bentoml.grpc.v1.service_pb2`?

Reply:
none, lets just do import then